### PR TITLE
[INGLONG-2754][Agent] Add stream metric data to Prometheus and JMX

### LINKED ISSUE #2754: [Feature][agent] add stream dimension metric data to Prometheus and JMX
### Description

add stream dimension metric data to Prometheus and JMX

### Use case

add stream dimension metric data to Prometheus and JMX

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by baomingyu**
Fixes #2754
add stream dimension metric data to Prometheus and JMX

### CODE REVIEW

File: inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/constant/CommonConstants.java [hunk 1]
```diff
@@ -137,5 +137,5 @@
     public static final String AGENT_NIX_OS = "nix";
     public static final String AGENT_NUX_OS = "nux";
     public static final String AGENT_COLON = ":";
-
+
```
Comment (healchow) [inline inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/constant/CommonConstants.java:140]: Why add those blanks?

File: inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sinks/AbstractSink.java [hunk 2]
```diff
@@ -42,4 +62,27 @@ public MessageFilter initMessageFilter(JobProfile jobConf) {
         return null;
     }
 
+    @Override
+    public void init(JobProfile jobConf) {
+
```
Comment (healchow) [inline inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sinks/AbstractSink.java:67]: Why add those empty line?

**Review by aloyszhang — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Review by healchow — APPROVED** [review]
LGTM

**Review by EMsnap — APPROVED** [review]
+1

**Pull request merged**